gui/src/main/java/DiceGameGUI.java

```diff
@@ -282,6 +282,17 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
     }// </editor-fold>//GEN-END:initComponents
 
     private void btnResetActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnResetActionPerformed
+        JOptionPane.showMessageDialog(
+                    null, 
+                    "Game Over! You save your score of " 
+                    + thisGame.getCurrentTotal()
+                    + "\nFinal Score: "
+                    + thisGame.getScore(),
+                    "Game Over!",
+                    JOptionPane.OK_OPTION);
+            btnRoll.setEnabled(false);
+            btnStop.setEnabled(false);
+        
```

Comment:
this is have conflict with my code in #119, I like your version better. Just throw it in an if statement if(thisGame.getCurrentTotal <23)